communities.json

```diff
@@ -742,6 +742,14 @@
       "quote": "We certainly think it's the most convenient platform for our communications.",
       "inviteCode": "tokio",
       "githubUrl": "https://github.com/tokio-rs"
+    },
+    {
+      "logo": "testinglibrary.png",
+      "title": "Testing Library",
+      "quote": "Simple and complete testing utilities that encourage good testing practices",
```

Comment:
This quote is intended for display as a comment on your Discord server, not a description of the library.

Reply:
Is it better now?